doc/structures/judgement.md

```diff
@@ -2,7 +2,7 @@
 # *Judgement* Object
 
 *Judgement* A judgement about the intent or nature of an observable.  For
-  example, is it malicious, meaning is is malware and subverts system
+  example, is it malicious, meaning is it malware and subverts system
```

Comment:
This content is generated with the command `lein doc`. Please fix the description in the source [file](https://github.com/threatgrid/ctim/blob/b18e471bd10186951a39a7769224b4c12f3f20b8/src/ctim/schemas/judgement.cljc#L24), then run `lein doc`.